### PR TITLE
Release 1.0.0.

### PR DESCRIPTION
**Pull request by shouze**

### CODE REVIEW

File: dcw [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env sh
+DCW_RELEASE="1.0.0"
+
+getopt -T &> /dev/null
+if [ $? -eq 0 ]
+then
+cat <<EOF
+
+You need to install gnu-getopt to run this program.
```
Comment (MikaFima) [inline dcw]: On linux it's named getopt, but on my distrib, it's available from scratch.. maybe not on all

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+    -s, --service          Name of docker compose service to run
+    -p, --service-ports    Run command with the service's ports enabled and mapped
+    -r,--run-options       Extra docker-compose run options (quote them)
+    -h, --help             Display this usage description
+    -v, --verbose          Display executed docker-compose command
+    -d, --dry-run          Display docker-compose command aims to be executed
+    -V, --version          Return dcw version
+
+  You can also configure following environment variables, either putting then
+  into a .dcw file in the current execution path of the command OR by
```
Comment (Spy-Seth) [inline README.md]: Could you provide an example of `.dcw` file?

File: dcw (hunk 1 above)
Comment (shouze) [inline dcw]: getopt is another tool, you should have getops installed too. when you run
```shell
$ getopts
```
on any Linux machine it should return something like that:
```shell
getopts: usage: getopts optstring name [arg]
```

File: README.md (hunk 2 above)
Comment (shouze) [inline README.md]: Yes of course, there's [some example in unit tests](https://github.com/rezzza/dcw/pull/1/files#diff-5daf4359b80aabad6123501ad97b946dR71) but ok that's not user friendly, I add it here too ;)

File: dcw [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+cat <<EOF
+
+Usage: dcw [OPTIONS] CMD
+
+  A docker compose wrapper to simplify everyday dev work with containers
+
+    -f, --file             Path to the docker compose file
+    -s, --service          Name of docker compose service to run
+    -p, --service-ports    Run command with the service's ports enabled and mapped
+    -r,--run-options       Extra docker-compose run options (quote them)
```
Comment (Spy-Seth) [inline dcw]: Missing space: `-r,--run-options`

Comment (shouze) [inline dcw]: good catch! :tiger:

**Pull request merged**